### PR TITLE
[Qt5] Fix the command line options when another instance is running

### PR DESCRIPTION
**Pull request by Chocobozzz**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -298,7 +298,7 @@ int main(int argc, char* argv[]) {
         qLog(Info)
             << "Clementine is already running - activating existing window";
       }
-      if (a.sendMessage(options.Serialize(), 5000)) {
+      if (a.sendMessage(QString::fromLatin1(options.Serialize().toBase64()), 5000)) {
```
Comment (hatstand) [inline src/main.cpp]: Probably better to just change the serialization format to base64

File: src/ui/mainwindow.cpp [hunk 2]
```diff
@@ -2047,13 +2047,13 @@ void MainWindow::PlaylistEditFinished(const QModelIndex& index) {
 
 void MainWindow::CommandlineOptionsReceived(
     const QString& serialized_options) {
-  if (serialized_options == "wake up!") {
+  if (QByteArray::fromBase64(serialized_options.toLatin1()) == "wake up!") {
```
Comment (Chocobozzz) [inline src/ui/mainwindow.cpp]: Not sure if I keep this part ?

**Comment by Chocobozzz** [comment]
Seems to be better like that

File: src/core/commandlineoptions.cpp [hunk 3]
```diff
@@ -293,7 +293,7 @@ bool CommandlineOptions::is_empty() const {
          toggle_pretty_osd_ == false && urls_.isEmpty();
 }
 
-QByteArray CommandlineOptions::Serialize() const {
+QString CommandlineOptions::Serialize() const {
```
Comment (hatstand) [inline src/core/commandlineoptions.cpp]: You can still use `QByteArray` everywhere I think

File: src/ui/mainwindow.cpp (hunk 2 above)
Comment (hatstand) [inline src/ui/mainwindow.cpp]: just remove it I think

**Comment by Chocobozzz** [comment]
Done, thanks for the review :)

**Pull request merged**